documents/moving-across-committees.md

```diff
@@ -0,0 +1,33 @@
+# Transfering a Working Group from the Node.js TSC to the Node.js Community Committee
+
+As interested, Working Groups that are focused on serving, growing, and building the community surrounding Node.js may decide that it would like to become a part of the Node.js Community Committee. Similarly, if a Working Group under the Node.js Community Committee is focused on more technical tasks and may want to transfer over to the TSC.
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## The Committee Transferring Checklist
+Here's a checklist for you to get started with transferring committees. This list is meant to help a Working Group make sure everything that _needs_ to be complete to effectively transfer committees actually _is_ complete. 
+
+- [] Ensure the Working Group is in alignment on moving from the current committee to the new committee
+- [] Have the currrent committee vote to decharter
+- [] Have the new committee vote (and approve) a new charter
+- [] Mark that the Working Group has been dechartered in the Working Groups documentation for the current committee
+- [] Submit a PR to the new committee adding the Working Group to the Working Groups documentation for the new committee
+
+## In depth: A guide to moving from one commmitte
+While the process of moving from one top level committee to another is likely going to be one of the smoother parts of this process, we've documented it in case you want some help with going through the process of this transition.
+
+### Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in the repository for the Working Group in question, proposing the move. Given the governance model, this will likely be a vote and will need majority approval from the Working Group's members.
```

Comment:
Funny thing is, it's actually pretty unlikely to be a vote. Charter changes are not called out specifically as requiring a vote, meaning it falls to the consensus-seeking model. Most issues resolved using the consensus seeking model don't go to a vote. That assumes that the WG uses the generic consensus seeking model though and that they copied that part of the charter from the charter template in the TSC repo, but it could be something else. We should leave that part out, as it's up to the WG to decide how they decide and vote on things.

I would reword this to say "Create an issue in the repository for the Working Group in question and gain approval from the working group for the move."